py/client/pydeephaven/table.py

```diff
@@ -72,6 +73,12 @@ def _parse_schema(self, schema_header):
         reader = pa.ipc.open_stream(schema_header)
         self.schema = reader.schema
 
+    @property
+    def meta_table(self) -> Table:
```

Comment:
Does this make sense as a `property`?

Reply:
I think you can argue either way. The server API also has it as a property.

Reply:
I'm not fully aware of the pythonic semantics one might assume when something is marked as a property. That said, naively I would be surprised as an end-user if I call a property method and it ends up needing to do a round-trip RPC.

Reply:
@property is the Pythonic way to implement a getter method, though not recommended for getters that take long time to finish.

Reply:
Isn't this a prime example of a method that might take a long time to finish?

Reply:
yes, and I guess it also depends on the real use case of it. OTOH we also want to keep the client API/server API consistent.